source/Command.cpp

```diff
@@ -62,7 +62,7 @@ const Command Command::BOARD(ONE << 11, "Board selected ship");
 const Command Command::HAIL(ONE << 12, "Talk to selected ship");
 const Command Command::SCAN(ONE << 13, "Scan selected ship");
 const Command Command::JUMP(ONE << 14, "Initiate hyperspace jump");
-const Command Command::FLEET_JUMP(ONE << 15, "");
+const Command Command::FLEET_JUMP(ONE << 15, "Fleet jump");
```

Comment:
I would prefer to use mOctave's suggested description, to match the standard jump command.
```suggestion
const Command Command::FLEET_JUMP(ONE << 15, "Initiate fleet jump");
```